git-some.sh

```diff
@@ -18,6 +18,12 @@ if grep --quiet --invert-match '^[[:digit:]]*$' <<< $commit_count; then
   exit 1
 fi
 
+COUNT=$(/bin/find . -maxdepth 1 -name "file-??.txt" | wc --lines)
```

Comment:
There's no need to use `/bin/find` when you may use `find` like you did with `wc` :-)